draft-ietf-taps-interface.md

```diff
@@ -2388,6 +2408,33 @@ The Capacity Profile for a selected protocol stack may be modified on a
 per-Message basis using the Transmission Profile Message Property; see
 {{send-profile}}.
 
+### Ojective of Using Multiple Paths {#multipath-objective}
+
+Name:
+: multipath-objective
+
+Type:
+: Enumeration
+
+Default:
+: Handover
+
+This property specifies the local objective of transferring data across multiple paths between the same end hosts if Parallel Use of Multiple Paths not set to Disabled (see {{multipath-mode}}). Possible values are:
+
+None:
+: Do not use other paths as long as the initial path is available, but allow the peer endpoint to make use it to achieve its objective.
+
+Handover:
+: The connection should only attempt to migrate between different paths when the original path is lost or becomes unreliable.
+The actual thresholds to declare a path unreliable are implementation specific.
+
+Interactive:
+: The connection should attempt to use multiple paths in parallel in order to minimize loss and delay. The actual strategy is implementation specific and may depend on the multipath protocol used, but should not aim to exceed the capacity provided by the best of the available paths.
+
+Aggregate:
+: The connection should attempt to use multiple paths in parallel in order to maximize bandwidth, possibly trading delay for bandwidth. The actual strategy is implementation specific.
```

Comment:
Not sure we need this half sentence about "trading delay for bandwidth". If the "needed" capacity is larger than what a single path can provide, there is no other chance than to use both. If you don't want that choose the "interactive" option instead